content/blog/2021/06/2021-06-17-libera-chat.adoc

```diff
@@ -0,0 +1,96 @@
+---
+:layout: post
+:title: "Jenkins IRC moves to Libera Chat"
+:tags:
+- community
+- announcement
+- chat
+authors:
+- markewaite
+- oleg_nenashev
+links:
+  discourse: https://community.jenkins.io/t/jenkins-irc-moved-to-libera-chat/146
+description: >
+  In this article we announce migration of the Jenkins community from Freenode IRC to Libera Chat.
+  See the blogpost for the migration details and next steps for our community.
+opengraph:
+  image: /images/post-images/2021-06-libera-chat/jenkins-irc-on-libera-chat-opengraph.png
+---
+
+image:/images/post-images/2021-06-libera-chat/jenkins-irc-on-libera-chat-opengraph.png[Jenkins IRC now on Libera Chat, role=center, link="/chat/#internet-relay-chat-irc"]
+
+We are happy to announce that
+the Jenkins community has moved all its link:/chat/#internet-relay-chat-irc[official IRC channels] to link:https://libera.chat/[Libera chat].
+We have started our migration on May 26 as a response to the 
+link:https://www.theregister.com/2021/05/26/freenode_irc_takeover/[hostile takeover]
+of hundreds of open source community channel by the new Freenode management.
+As decided  by the Jenkins governance meeting on June 16th, 
+Libera Chat IRC channels are the only official channels going forward.
+
+A more detailed history of the transition is available on the link:https://groups.google.com/g/jenkinsci-dev/c/pEcun5ZSfgo/m/ztRg03VNAgAJ[Jenkins developers mailing list].
+
+=== What did we move?
+
+We have created the following IRC chats:
+
+that currently live on freenode to libera.chat:
+
+* `#jenkins` - general discussion channel for Jenkins users and developers.
+  We no longer consider it a central chat for all things Jenkins, but we intend to keep it available to users
+  who want to keep using IRC.
+* `#jenkins-hosting` - Chat used by the link:/project/teams/hosting/[Hosting Team] to manage hosting automation
+* `#jenkins-infra` - Chat used by the link:/projects/infrastructure/#team[Infrastructure Team] to coordinate efforts related to Jenkins project infrastructure and the incident response
+* `#jenkins-release`- Chat used for Jenkins core release coordination by the link:https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#team[Release Team]
+
+All other channels have NOT been moved to Libera Chat IRC and were replaced by other channels.
+Most notable channels:
+
+* `#jenkins-meeting` - the link:/project/governance-meeting/[Jenkins Governance Meeting] uses combination of Zoom and Google Docs at the moment. We do not longer use the IRC chat for that purpose.
+* `#jenkins-cert` - no private chat going forward. The link:/security[Security Team] will use mailing lists only going forward
```

Comment:
As this was never a publicly documented or accessible channel, there is no reason to bring it up here.

Reply:
I would keep it if you do not mind. I would rather be transparent that we had this channel and that it might have been compromised

Reply:
That's not what this says

Also, nobody without access has a chance to understand what it means; not to mention we checked already for private content.

So, hard 👎 from me for this.

Reply:
@daniel-beck just FYI this has already been merged, so a followup PR needs to be done.

I agree though, there's no point in listing a channel that wasn't documented before. I believe it was also a private channel wasn't it? There's no logs so there's no security concerns. I think it just makes things confusing to list a channel nobody knows about.

On the other hand, I'm not sure we need a list for one channel. Jenkins meeting has been dead for more than a year, I think its safe to drop this section, but I'm also not motivated enough to bother with a new PR.

Reply:
> FYI this has already been merged, so a followup PR needs to be done.

Thanks, I noticed, doesn't mean whoever messed this up shouldn't clean it up though.

Very disappointing to see we're now spreading Jenkins FUD ourselves for no reason at all.